content/en/blog/_posts/2020-12-10-Kubernetes Volume Snapshot Moves to GA.md

```diff
@@ -0,0 +1,219 @@
+
+## What are the limitations?
+
+The GA implementation of volume snapshots for Kubernetes has the following limitations:
+
+- Does not support reverting an existing PVC to an earlier state represented by a snapshot (only supports provisioning a new volume from a snapshot).
+
+### How to learn more?
+
+The code repository for snapshot APIs and controller is here: https://github.com/kubernetes-csi/external-snapshotter
+
+Check out additional documentation on the snapshot feature here: http://k8s.io/docs/concepts/storage/volume-snapshots and https://kubernetes-csi.github.io/docs/
+
+## How to get involved?
+
+This project, like all of Kubernetes, is the result of hard work by many contributors from diverse backgrounds working together.
+
+We offer a huge thank you to the contributors who stepped up these last few quarters to help the project reach GA. We want to thank Saad Ali, Michelle Au, Tim Hockin, and Jordan Liggitt for their insightful reviews and thorough consideration with the design, thank Andi Li for his work on adding the support of the snapshot validation webhook, thank Grant Griffiths on implementing metrics support in the snapshot controller and handling password rotation in the validation webhook, thank Chris Henzie, Raunak Shah, and Manohar Reddy for writing critical e2e tests to meet the scalability and stability requirements for graduation, thank Kartik Sharma for moving snapshot APIs and client lib to a separate go module, and thank Raunak Shah and Prafull Ladha for their help with upgrade testing from beta to GA.
+
+There are many more people who have helped to move the snapshot feature from beta to GA. We want to thank everyone who has contributed to this effort:
+- [Andi Li](@AndiLi99)
+- [Ben Swartzlander](@bswartz)
+- [Chris Henzie](@chrishenzie)
+- [Christian Huffman](@huffmanca)
+- [Grant Griffiths](@ggriffiths)
+- [Humble Devassy Chirammal](@humblec)
+- [Jan Šafránek](@jsafrane)
+- [Jiawei Wang](@jiawei0277)
+- [Jing Xu](@jingxu97)
+- [Jordan Liggitt](@liggitt)
+- [Kartik Sharma](@Kartik494)
+- [Madhu Rajanna](@Madhu-1)
+- [Manohar Reddy](@boddumanohar)
+- [Michelle Au](@msau42)
+- [Patrick Ohly](@pohly)
+- [Prafull Ladha](@prafull01)
+- [Prateek Pandey](@prateekpandey14)
+- [Raunak Shah](@RaunakShah)
+- [Saad Ali](@saadali)
+- [Saikat Roychowdhury](@saikat-royc)
+- [Tim Hockin](@thockin)
+- [Xiangqian Yu](@yuxiangqian)
+- [Xing Yang](@xing-yang)
+- [Zhu Can](@zhucan)
+
+For those interested in getting involved with the design and development of CSI or any part of the Kubernetes Storage system, join the [Kubernetes Storage Special Interest Group (SIG)](https://github.com/kubernetes/community/tree/master/sig-storage). We’re rapidly growing and always welcome new contributors.
```

Comment:
nit:
```suggestion
For those interested in getting involved with the design and development of CSI or any part of the Kubernetes Storage system, join the [Kubernetes Storage Special Interest Group](https://github.com/kubernetes/community/tree/master/sig-storage)  (SIG). We’re rapidly growing and always welcome new contributors.
```

Reply:
Addressed this in the latest commit, PTAL.